gruntfileConfig.json

```diff
@@ -4,6 +4,7 @@
   "src": "./src",
   "assets": "./src/assets",
   "theme": "./node_modules/patternpack-example-theme",
+  "logo": "/theme-assets/images/pp-logo.svg",
```

Comment:
> If you are authoring the theme, because you hardcoded "logo": "/theme-assets/images/pp-logo.svg", into the configuration. All themes will by default use pp-logo.svg as their default logo name. If I don't want each pattern library to have to configure a special name, I (as a theme author) must use pp-logo.svg as my logo name, so that the patternpack defaults will work.

Just making sure I understand - you're talking about someone authoring their own theme, right? I see two options in this case:

1. This helper is merely making `{{logo}}` available to the theme. If you're writing your own theme, you can hard code your own logo path
2. Or you *could* use the `{{logo}}` helper in your theme Handlebars file, and then configure it using `options.theme`, but that would be redundant. If you're building your own theme, just leave it blank.

> One other thing. The default values should be placed in BOTH gruntfileConfig.json and in the defaults section of tasks/patternpack.js

I see the default value in this diff in `patternpack.js`, did I miss something? It's on the new line 24 in there.